lib/allowed-features.js

```diff
@@ -79,6 +79,14 @@ export const isFeatureEnabled = (collective, feature) => {
   return featureStatus === 'ACTIVE' || featureStatus === 'AVAILABLE';
 };
 
+/**
+ * Returns true if the feature is DISABLED and the account is using a new plan
+ */
+export const requiresUpgrade = (collective, feature) => {
+  const featureStatus = getFeatureStatus(collective, feature);
+  return featureStatus === 'DISABLED' && collective.platformSubscription;
+};
+
```

Comment:
A new check to determine if we should show the Upgrade CTA, meaning that it will never be used for features that are "UNSUPPORTED", this is instead handled by the Dashboard Menu that should not show links to unsupported features